x-pack/legacy/plugins/reporting/export_types/common/execute_job/get_conditional_headers.test.ts

```diff
@@ -26,15 +28,7 @@ describe('conditions', () => {
     };
 
     const { conditionalHeaders } = await getConditionalHeaders({
-      job: {
-        title: 'cool-job-bro',
-        type: 'csv',
-        jobParams: {
-          savedObjectId: 'abc-123',
-          isImmediate: false,
-          savedObjectType: 'search',
-        },
```

Comment:
A lot of the tests had mock data like this: job params that don't line up to existing features. The `csv` export type doesn't use savedObjectId and savedObjectType as jobParams. See `JobParamsDiscoverCsv` is the actual definition. Invalid mocks were part of the reason Typescripting is taking awhile.

Reply:
Apologies on that -- I believe I wrote these a while ago. You're dead-on about TS taking so long: I believe I originally wrote these mocks with little TS input, so they were likely incorrect. In any case, sorry!

Reply:
Ha, it's not problem. Mostly I write these things down because I learn as I go